leaf-ui/scripts/main.js

```diff
@@ -147,10 +147,16 @@ if (document.cookie){
 			break;
 		}
 	}
-	console.log(graph.fromJSON(JSON.parse(prevgraph)));
+
 	if (prevgraph){
-		graph.fromJSON(JSON.parse(prevgraph));
+		try {
```

Comment:
Using try/catch as a sanity check.

Reply:
Actually, I think it can be fixed in the if statement.
`
if (cookies[i].indexOf("graph=") >= 0){
			prevgraph = cookies[i].substr(6);
`
It is assuming the substring "graph=" is in the beginning of the string. But if it is not then the JSON string is not properly formatted. I propose this instead:

`
if (cookies[i].indexOf("graph=") != 0){
			prevgraph = cookies[i].substr(cookies[i].indexOf("graph=") + 6);
`